Simplified/Accounts.swift

```diff
@@ -4,12 +4,23 @@ let currentAccountIdentifierKey  = "NYPLCurrentAccountIdentifier"
 let userAboveAgeKey              = "NYPLSettingsUserAboveAgeKey"
 let userAcceptedEULAKey          = "NYPLSettingsUserAcceptedEULA"
 let accountSyncEnabledKey        = "NYPLAccountSyncEnabledKey"
+let betaUrl = URL(string: "https://libraryregistry.librarysimplified.org/libraries/qa")!
+let prodUrl = URL(string: "https://libraryregistry.librarysimplified.org/libraries")!
+let betaUrlHash = betaUrl.absoluteString.md5().base64EncodedStringUrlSafe().trimmingCharacters(in: ["="])
+let prodUrlHash = prodUrl.absoluteString.md5().base64EncodedStringUrlSafe().trimmingCharacters(in: ["="])
 
```

Comment:
Comments on what this functions accomplishes and description of the parameters would be nice :-)